### PR TITLE
Download libgccjit-12-dev package in the CI

### LINKED ISSUE #173: Use PPA in the libgccjit12 CI
https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test

When it's added.

### PR DESCRIPTION
**Pull request by antoyo**
Fix #173

### CODE REVIEW

**Comment by antoyo** [comment]
It seems like Ubuntu stable has an old build of libgccjit, so wait until it's updated.

**Comment by antoyo** [comment]
We'll possibly be able [to have gcc 12 soon](https://github.com/actions/runner-images/issues/6509).

**Comment by Zopolis4** [comment]
GCC 12 was [added](https://github.com/actions/runner-images/commit/042094639142bab5548109945dc260e5ec002ad0) a few days ago.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.14 # Switch back to ubuntu-latest when it is 22.04
```
Comment (GuillaumeGomez) [inline .github/workflows/ci.yml:16]: 22.14, you sure? ;)

**Pull request merged**